Package.swift

```diff
@@ -72,6 +72,11 @@ let package = Package(
             targets: ["PackageDescription"]
         ),
     ],
+    dependencies: [
```

Comment:
We're probably going to need to handle these dependencies as we do with the others, at the bottom of the manifest.  The reason is that in some cases the build needs to be run with all the dependencies checked out next to SwiftPM.  I will take a look at what's needed here based on how it's being built.